### ISSUE TITLE
Keep import package list in glide.yaml alphabetical 

### ISSUE THREAD

**Issue by mhemmings** [Masterminds/glide#454]
When the first glide.yaml is generated, the dependencies are listed in alphabetical order.

However, future dependencies added with `glide get` are just appended to the end. It would be nice to keep them in order.

Happy to PR in the change if it is of interest to others?

**Comment by mattfarina** [comment]
@mhemmings We need to hold off on doing this right now because of one reason. Order matters. The first reference to a dependency and version wins right now. That means you can alter the order in the `glide.yaml` file to have a different version win.

This is especially true when importing a dependency of a dependency.

@sdboyer is working on a smarter resolver. Once that's done we can, hopefully, start keeping track of things in alpha order.

This is a good question and I hope we can get there someday.

**Comment by sdboyer** [comment]
indeed, once we have that new system in place, ensuring alpha order on the deps in glide.yaml will be totally feasible; it will have no bearing on the outcome.

OTOH, there's also a school of thought here that says it's better to preserve the original order the user declared, so as to avoid meaningless churn in the file itself. i don't really have an opinion as to whether one or the other is really better.

**Comment by vearutop** [comment]
Alphabetical order is very helpful when resolving conflicts in two `glide.yaml`. The diff could be very messy and unclear without ordering.
